conwaywebclient/ConwayClient.cs

```diff
@@ -0,0 +1,71 @@
+using System;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using ConwayLib;
+using ConwayWebModel;
+using System.Text.Json;
+using static System.Math;
+using System.Net.Http.Json;
+using System.Collections.Generic;
+
+namespace ConwayWebClient
+{
+    public static class ConwayClient
+    {
+        static HttpClient mClient = new HttpClient();
+        const string ROOT = "Board/";
+        static string mBaseAddress;
+        
+        public static async Task<IEnumerable<BoardInfo>> GetBoardsAsync()
+        {
+            IEnumerable<BoardInfo> boards = null;
+            HttpResponseMessage response = await mClient.GetAsync(ROOT);
+            if (response.IsSuccessStatusCode)
+            {
+                boards = await JsonSerializer.DeserializeAsync<IEnumerable<BoardInfo>>(await response.Content.ReadAsStreamAsync());
+            }
+            return boards;
+        }
+        public static async Task<BoardDetail> GetBoardDetailAsync(int id)
+        {
+            return await GetBoardDetailAsync(ROOT + id.ToString());
+        }
+        
+        public static async Task<BoardDetail> GetBoardDetailAsync(string url)
+        {
+            BoardDetail state = null;
+            HttpResponseMessage response = await mClient.GetAsync(url);
+            if (response.IsSuccessStatusCode)
+            {
+                state = await response.Content.ReadAsAsync<BoardDetail>();
+            }
+            return state;
```

Comment:
Do you really want to return null for a bad situation? Will the caller actually cope with this? Would an exception be better?